.github/workflows/cloudflare.yml

```diff
@@ -56,8 +56,6 @@ jobs:
           restore-keys: ${{ runner.os }}-${{ hashFiles('**/yarn.lock') }}
       - name: Yarn Install
         run: yarn install --immutable
-      - name: Build Packages
```

Comment:
noticed a redundant build packages that can save us a good minute

Reply:
🐐